storm/check.py

```diff
@@ -0,0 +1,573 @@
+            return {}
+
+    def get_storm_cluster_summary(self):
+        """ Make the storm cluster summary metric request.
+
+        :return: Cluster Summary Stats Response
+        :rtype: dict
+        """
+        return self.get_request_json("/api/v1/cluster/summary", "Error retrieving Storm Cluster Summary")
+
+    def get_storm_nimbus_summary(self):
+        """ Make the storm nimbus summary metric request.
+
+        :return: Nimbus Summary Stats Response
+        :rtype: dict
+        """
+        return self.get_request_json("/api/v1/nimbus/summary", "Error retrieving Storm Nimbus Summary")
+
+    def get_storm_supervisor_summary(self):
+        """ Make the storm supervisor summary metric request.
+
+        :return: Supervisor Summary Stats Response
+        :rtype: dict
+        """
+        return self.get_request_json("/api/v1/supervisor/summary", "Error retrieving Storm Supervisor Summary")
+
+    def get_storm_topology_summary(self):
+        """ Make the storm topology summary metric request.
+
+        :return: Topology Summary Stats Response
+        :rtype: dict
+        """
+        return self.get_request_json("/api/v1/topology/summary", "Error retrieving Storm Topology Summary")
+
+    def get_topology_info(self, topology_id, interval=60):
+        """ Make the topology info metric request.
+
+        :param topology_id: Topology Id
+        :type topology_id: str
+        :return: Topology Info Response
+        :rtype: dict
+        """
+        params = {'window': interval}
+        return self.get_request_json("/api/v1/topology/{}".format(topology_id),
+                                     "Error retrieving Storm Topology Info for topology:{}".format(topology_id),
+                                     params=params)
+
+    def get_topology_metrics(self, topology_id, interval=60):
+        """ Make the storm topology metrics request.
+
+        :param topology_id: Topology Id
+        :type topology_id: str
+        :param interval: Interval in seconds
+        :type interval: int|long
+        :return: Topology Metrics Stats Response
+        :rtype: dict
+        """
+
+        params = {'window': interval}
+        return self.get_request_json("/api/v1/topology/{}/metrics".format(topology_id),
+                                     "Error retrieving Storm Topology Metrics for topology:{}".format(topology_id),
+                                     params=params)
+
+    def process_cluster_stats(self, environment, cluster_stats):
+        """ Process Cluster Stats Response
+
+        :param environment: Storm Environment
+        :type environment: str
+        :param cluster_stats: Cluster stats response
+        :type cluster_stats: dict
+        :return: Extracted cluster stats metrics
+        :rtype: dict
+        """
+        if len(cluster_stats) == 0:
+            return {}
+        else:
+            version = _g(cluster_stats, 'unknown', None, 'version').replace(' ', '_').lower()
```

Comment:
I think using `_g` here is unnecessary and should be replaced by:

```python
cluster_stats.get('version', 'unknown')
```
This make it easier to read, a little better performance-wise and more pythonic for contributors.
We should do the same for every other use of `_g` where the function is `None` and `components` is a single string.